lang/cs/Org.Apache.REEF.IMRU.Tests/TestEvaluatorManager.cs

```diff
@@ -129,7 +129,7 @@ public void TestResetFailedEvaluators()
         [Fact]
         public void TestReachedMaximumNumberOfEvaluatorFailures()
         {
-            var evalutorManager = CreateTestEvaluators(3, 2);
+            var evalutorManager = CreateTestEvaluators(3, 1);
```

Comment:
Why 1 instead of 2 ? Also, is it actually `evaluatorManager` (with 'a' after 'u') or does this name shadow sth ?